docker/build.sh

```diff
@@ -1,5 +1,12 @@
 export JETBOT_VERSION=jp44
 
+DOCKER_JSON_FILE="/etc/docker/daemon.json"
```

Comment:
Cool, this is handy.

It may help to put this in a separate script, since it's needed for building the containers independently.  
Also it would be great to have a little section in documentation explaining what this does, since it's so commonly needed if you're working with Docker on Jetson.  So much so that I'm surprised it's not default actually.

---

Step 1 - Configure Docker Runtime

If you haven't already, call the ``set_nvidia_runtime.sh`` script to configure the default docker runtime to NVIDIA.  This adds the line ``default-runtime: nvidia`` to ``/etc/docker/daemon.json`` and restarts the docker daemon.  This enables the use of GPU related components during the docker build and run phases, which is necessary for building the JetBot containers.

```
cd docker
./set_nvidia_runtime.sh
```

Reply:
Makes sense.
I separated that out as `set_nvidia_runtime.sh` now.